### ISSUE TITLE
Airtable Form: 2022-09-11

### ISSUE THREAD

**Issue by marcusrelacion** [AIGAdc/dcdesignweek.org-11ty#87]
Reporter:
Ariana

Date:
2022-09-11

Page URL:
https://deploy-preview-62--2022-dcdw.netlify.app/accessibility/

Feedback:
First section of text in white section should be changed so "This accessibility statement aims to:" is the header in the larger black text. Then the list of items can go after the short rule line.
